src/App.tsx

```diff
@@ -1,34 +1,21 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import './App.css'
+import "./App.css";
+import Navbar from "./components/Navbar/Navbar";
+import BooksPage from "./domain/books/BooksPage";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import { CartPage } from "./domain/cart/CartPage";
 
 function App() {
-  const [count, setCount] = useState(0)
-
   return (
     <div className="App">
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src="/vite.svg" className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://reactjs.org" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
+      <BrowserRouter>
+        <Navbar />
+        <Routes>
+          <Route path="/" element={<BooksPage />} />
+          <Route path="/Cart" element={<CartPage />} />
+        </Routes>
+      </BrowserRouter>
```

Comment:
We should always write our routes in a separate Routes file and then import them here.
It alright for now but please keep it in mind for future.

We should always maintain our folder structure regardless of the size of the app.

Reply:
I will keep that  in mind for sure!